### PR TITLE
@grafana/toolkit: cleanup

### PR DESCRIPTION
**Pull request by stevenvachon**
The breakdown can be seen by looking at the separate commit messages.

This is in support of #21006.

### CODE REVIEW

**Comment by ryantxu** [comment]
this is looking good

File: packages/grafana-toolkit/src/cli/tasks/plugin.build.ts [hunk 1]
```diff
@@ -1,154 +1,155 @@
...
 // @ts-ignore
 const typecheckPlugin = useSpinner<void>('Typechecking', async () => {
   await execa('tsc', ['--noEmit']);
 });
 
-const getTypescriptSources = () => {
-  const globPattern = path.resolve(process.cwd(), 'src/**/*.+(ts|tsx)');
-  return glob.sync(globPattern);
-};
+const getTypescriptSources = () => globby(resolvePath(process.cwd(), 'src/**/*.+(ts|tsx)'));
```
Comment (dprokop) [inline packages/grafana-toolkit/src/cli/tasks/plugin.build.ts]: What are globby advantages over glob?

**Review by dprokop — APPROVED** [review]
Looking good! Thanks @stevenvachon

File: packages/grafana-toolkit/src/cli/tasks/plugin.build.ts (hunk 1 above)
Comment (stevenvachon) [reply packages/grafana-toolkit/src/cli/tasks/plugin.build.ts]: Built-in support for promises. Also, multiple patterns per function call.

**Review by ryantxu — APPROVED** [review]
LGTM

**Pull request merged**